public/data/vivekpal1.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Vivek Pal",
+  "type": "personal",
+  "bio": "Love to Learn n Explore!",
+  "avatar": "https://github.com/vivekpal1.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/vivekpal1",
+      "icon": "github"
+    },
+    {
+      "name": "My Website",
+      "url": "https://vivekpal.in/",
+      "icon": "globe"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vivekpal_",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Open Source",
+      "date": "February 2022",
+      "icon": "github",
+      "color": "Orange",
+      "description": "",
+      "url": "https://github.com/vivekpal1"
+    },
+    {
+      "title": "Started Web Development",
+      "date": "June 2021",
+      "icon": "earch",
```

Comment:
⚠  `earch` is the wrong keyword, you can use `globe` instead.

 you should check [Icon.md](https://github.com/EddieHubCommunity/LinkFree/blob/main/icons.md) for more icons. 😉

```suggestion
      "icon": "globe",
```